.vitepress/theme/index.ts

```diff
@@ -1,12 +1,16 @@
 import type { Theme } from 'vitepress';
 import EncryptedSolution from '../../src/components/EncryptedSolution.vue';
+import MaterialFAQ from '../../src/components/MaterialFAQ.vue';
 import Layout from './Layout.vue';
 import './style/style.scss';
 
 export default {
     Layout,
     enhanceApp({ app }) {
         // make the commonly used components available everywhere
-        app.component('EncryptedSolution', EncryptedSolution);
+        app.component('EncryptedSolution', EncryptedSolution).component(
```

Comment:
I'd appreciate a formatted version like the following:

```typescript
app
  .component('EncryptedSolution', EncryptedSolution)
  .component('MaterialFAQ', MaterialFAQ);
```

Reply:
This is not so easily possible using prettier which is rather opinionated (which is great!)